src/test/java/teammates/test/cases/webapi/GetFeedbackSessionsActionTest.java

```diff
@@ -20,16 +24,126 @@ protected String getRequestMethod() {
         return GET;
     }
 
-    @Test
     @Override
     protected void testExecute() throws Exception {
-        // TODO
+        // see individual tests
+    }
+
+    @Test
+    protected void testExecute_asInstructorWithCourseId_shouldReturnAllSessionsForCourse() {
+        InstructorAttributes instructor1OfCourse1 = typicalBundle.instructors.get("instructor1OfCourse1");
+        loginAsInstructor(instructor1OfCourse1.googleId);
+
+        String[] submissionParam = {
+                Const.ParamsNames.COURSE_ID, instructor1OfCourse1.getCourseId(),
+        };
+
+        GetFeedbackSessionsAction a = getAction(submissionParam);
+        FeedbackSessionsData fsData = (FeedbackSessionsData) getJsonResult(a).getOutput();
+
+        assertEquals(6, fsData.getFeedbackSessions().size());
```

Comment:
You should verify that there are 6 feedback sessions associated with the course. Ideally, maybe also verify that the feedback sessions actually match?